wings/1.0/configuration.md

```diff
@@ -29,3 +29,24 @@ docker:
       username: "registryusername"
       password: "registrypassword"
 ```
+
+
+## Mount Points
+
+These settings can be used to allow a mounting point on the machine.
+This configuration is mandatory to allow panel users to mount a mount point.
+
+### Available Keys
+
+| Setting Key | Default Value | Notes |
+|-------------|---------------|-------|
+| allowed_mounts        | empty array          | List of fullpath |
```

Comment:
`List of fullpath` I would say `List of avaible mounts`

Reply:
The `allowed_mounts` list is prefix based; if you have `/` in the list, it would allow all host directories (including `/` itself) to be mounted to a server (other than [the directories explicitly denied by the panel](https://github.com/pterodactyl/panel/blob/develop/app/Models/Mount.php#L94-L98)).

Reply:
So Avaible Mount points would be best woudn't it

Reply:
`Avaible mount prefixes`

Reply:
Fixed